### DISCUSSION TITLE
add vtracer command text with command arguments based on slider parameters change

### DISCUSSION THREAD

**Discussion by wis** [visioncortex/vtracer#82] [Ideas]
I tried to add this myself and create a PR, but it seems like the web app in the repo does not look like the same web app on [the website ](https://www.visioncortex.org/vtracer)

the website's web app:
![image](https://github.com/visioncortex/vtracer/assets/9993663/89040996-bdc0-44c7-892a-43776c43954b)

the web app in the repo:
![image](https://github.com/visioncortex/vtracer/assets/9993663/00957b8b-a6ce-48ba-a015-259b2cc71d6a)

I followed instructions in the readme.md, `wasm-pack build`, `npm install`, `npm run start` etc, and opened localhost:8080

Concept design:
![image](https://github.com/visioncortex/vtracer/assets/9993663/6408e3ea-e62c-4b1e-a79c-28de80b966f4)

**Comment by tyt2y3** [comment]
The only difference is the CSS haha!
